main.py

```diff
@@ -368,8 +378,8 @@ async def on_connect():
 @wait_ready(bot=bot)
 @event_check(lambda m: not bot.tester or bot.sync_is_owner(m.author))
 async def on_message(message):
-    if re.fullmatch(r"<@!?661466532605460530>", message.content):
-        await message.channel.send(f"My prefix is `{await bot.get_prefix(message)}`")
+    if re.fullmatch(rf"<@!?{bot.user.id}>", message.content):
+        await message.channel.send(f"My prefix is `{await bot._prefix_for_message(message)}`")
```

Comment:
again why is it using a different method for no reason

Reply:
this avoids testing message. you just want to fetch relevant prefix